cgmes/cgmes-conversion/src/test/resources/logback-test.xml

```diff
@@ -27,4 +27,12 @@
         level="info" />
     <!-- To watch the properties of every object converted, level to debug -->
     <logger name="com.powsybl.cgmes.conversion.Conversion" level="info" />
+        <!-- To watch the Hvdc configurations after the topological analysis, level to debug -->
+    <logger name="com.powsybl.cgmes.conversion.elements.hvdc.CgmesDcConversion" level="info" />
+    <logger name="com.powsybl.cgmes.conversion.elements.hvdc.Adjacency" level="info" />
+    <logger name="com.powsybl.cgmes.conversion.elements.hvdc.TPnodeEquipments" level="info" />
+    <logger name="com.powsybl.cgmes.conversion.elements.hvdc.Islands" level="info" />
+    <logger name="com.powsybl.cgmes.conversion.elements.hvdc.IslandsEnds" level="info" />
+    <logger name="com.powsybl.cgmes.conversion.elements.hvdc.IslandEndHvdc" level="info" />
+    <logger name="com.powsybl.cgmes.conversion.elements.hvdc.Hvdc" level="info" />
```

Comment:
More of a general remark but why is the test logback level at info?

Reply:
should be set to `DEBUG` to enforce the coverage of the debug sections in the analysis of the topology.

Reply:
Done